addons/website/views/snippets/s_media_list.xml

```diff
@@ -2,57 +2,73 @@
-                </div>
-            </div>
-            <div class="row" data-name="Media item">
-                <div class="col-lg-4">
-                    <img class="img img-fluid" src="/web/image/website.s_media_list_default_image_3" alt=""/>
-                </div>
-                <div class="col-lg-8 s_media_list_body">
-                    <h3>Post heading</h3>
-                    <span class="s_badge badge bg-gamma" data-name="Badge">
-                        <i class="fa fa-1x fa-fw fa-clock-o"/>3 months ago
-                    </span>
-                    <span class="s_badge badge bg-gamma" data-name="Badge">
-                        <i class="fa fa-1x fa-fw fa-user"/>by John DOE
-                    </span>
-                    <p>Use this component for creating a list of featured elements to which you want to bring attention.</p>
-                    <p><a href="#">Continue reading <i class="fa fa-long-arrow-right align-middle ml-1"/></a></p>
+                <div class="s_media_list_col col-lg-12 s_col_no_bgcolor pt16 pb16" data-name="Media item">
+                    <div class="s_media_list_item s_media_list_item_3 bg-white align-items-center">
+                        <div class="s_media_list_img_wrapper">
+                            <img src="/web/image/website.s_media_list_default_image_3" class="s_media_list_img" alt=""/>
+                        </div>
+                        <div class="s_media_list_body">
+                            <h3>Post heading</h3>
+                            <p>Use this component for creating a list of featured elements to which you want to bring attention.</p>
+                            <a href="#">Continue reading <i class="fa fa-long-arrow-right align-middle ml-1"/></a>
+                        </div>
+                    </div>
                 </div>
             </div>
         </div>
     </section>
 </template>
 
 <template id="s_media_list_options" inherit_id="website.snippet_options">
-    <xpath expr="//div[@data-js='layout_column']" position="attributes">
-        <attribute name="data-exclude" add=".s_media_list" separator=","/>
-    </xpath>
     <xpath expr="." position="inside">
-        <div data-selector=".s_media_list > * > .row" data-drop-near=".s_media_list > * > .row"/>
-    </xpath>
-    <xpath expr="//div[@data-js='SnippetMove'][contains(@data-selector,'section')]" position="attributes">
-        <attribute name="data-selector" add=".s_media_list > * > .row" separator=","/>
+        <div data-js="Box" data-selector=".s_media_list_col" data-target=".s_media_list_item">
+            <we-select string="Layout">
+                <we-button data-select-class="s_media_list_item_2">1/2 - 1/2</we-button>
+                <we-button data-select-class="s_media_list_item_3">1/3 - 2/3</we-button>
+                <we-button data-select-class="s_media_list_item_1">1/4 - 3/4</we-button>
+            </we-select>
```

Comment:
- It's very difficult to select and move a media item.
- It's very difficult to select the column containing the image.
- The options are messy and not necessary well applied. 
- ...
[Part of the spec](https://www.awesomescreenshot.com/image/4511892/f4ddf93668ce12f74ac3031ddf29c58a)

@sbuhl @qsm-odoo 
So, what should I do?

Indeed, if we keep it that way, it needs a fix for the mobile part. 👍

Reply:
Sorry for my late answer @Cocographique but I am kinda receiving a mail per second right now ^^ Need a decision from @sbuhl but in any case, it will require to restore a grid system for this so you can already put that back ;)